pkg/repository/maintenance/maintenance.go

```diff
@@ -431,6 +431,12 @@ func buildJob(cli client.Client, ctx context.Context, repo *velerov1api.BackupRe
 	// Get the service account from the Velero server deployment
 	serviceAccount := veleroutil.GetServiceAccountFromVeleroServer(deployment)
 
+	// Get the security context from the Velero server deployment
+	securityContext := veleroutil.GetSecurityContextsFromVeleroServer(deployment)
```

Comment:
nit
```suggestion
	securityContext := veleroutil.GetContainerSecurityContextsFromVeleroServer(deployment)
```

Reply:
I won't fix this in the next days as I'm on vacation.